packages/superset-ui-legacy-preset-chart-deckgl/src/layers/Hex/controlPanel.js

```diff
@@ -17,63 +17,64 @@
  * under the License.
  */
 import { sections } from '@superset-ui/chart-controls';
-import { t, legacyValidateInteger, isFeatureEnabled, FeatureFlag } from '@superset-ui/core';
+import { t } from '@superset-ui/core';
 import { formatSelectOptions } from '../../utilities/utils';
 import {
+  autozoom,
+  extruded,
   filterNulls,
+  gridSize,
   jsColumns,
   jsDataMutator,
-  jsTooltip,
   jsOnclickHref,
-  fillColorPicker,
-  strokeColorPicker,
-  filled,
-  stroked,
-  extruded,
-  viewport,
+  jsTooltip,
   mapboxStyle,
-  geojsonColumn,
+  spatial,
+  viewport,
 } from '../../utilities/Shared_DeckGL';
-import { dndGeojsonColumn } from '../../utilities/sharedDndControls';
-
-const geojson = isFeatureEnabled(FeatureFlag.ENABLE_EXPLORE_DRAG_AND_DROP)
-  ? dndGeojsonColumn
-  : geojsonColumn;
 
 export default {
   controlPanelSections: [
     sections.legacyRegularTime,
     {
       label: t('Query'),
       expanded: true,
-      controlSetRows: [[geojson], ['row_limit'], [filterNulls], ['adhoc_filters']],
+      controlSetRows: [[spatial], ['size'], ['row_limit'], [filterNulls], ['adhoc_filters']],
     },
     {
       label: t('Map'),
       controlSetRows: [
         [mapboxStyle, viewport],
-        // TODO [autozoom, null], // import { autozoom } from './Shared_DeckGL'
-      ],
-    },
-    {
-      label: t('GeoJson Settings'),
-      controlSetRows: [
-        [fillColorPicker, strokeColorPicker],
-        [filled, stroked],
-        [extruded, null],
+        ['color_picker'],
+        [autozoom],
```

Comment:
I decided to add the autozoom control as it appears to be used elsewhere, too. To make the behavior unchanged in the example chart, I'm adding `autozoom: false` to the chart metadata in the PR that bumps the version of this plugin on the main repo (forthcoming).